packages/trader/src/Modules/CFD/Components/cfd-personal-details-form.tsx

```diff
@@ -1,6 +1,7 @@
-import { Field, Formik } from 'formik';
-import PropTypes from 'prop-types';
+import { Field, FieldProps, Formik, FormikHelpers as FormikActions, FormikProps } from 'formik';
 import React from 'react';
+// @ts-ignore
```

Comment:
I think we can add `declare module '@deriv/account'` to globals.d.ts file. That should ignore this.
Please make sure to other mono repo packages too